app/components/Hero.tsx

```diff
@@ -0,0 +1,43 @@
+// LIBRARY IMPORTS
+import { Box, Heading } from "@chakra-ui/react";
+import Image from "next/image";
+
+// LOCAL IMPORTS
+import heroImage from "../../public/images/hero-photo.png";
+
+// HERO COMPONENT DEFINITION
+export default function Hero() {
+  return (
+    <Box
+      position="relative"
+      textAlign="center"
+      bg="white"
+      minH="calc(38.25vw + 6rem)"
```

Comment:
Do you mind sharing how did you get to these numbers? Teach me your ways :)

Reply:
Me too!

Reply:
This was some straight up chatGPT insight! The outermost Box component wasn't consistently tall enough across various screen sizes to show the white space. I didn't want to set a fixed height but I  couldn't quite figure out how to keep it proportionate to the image and heading.
The `calc()` expression calculates the height based on the aspect ratio of the image and adds the height of the inner Box. The aspect ratio of the image is 610 / 1728. To get the height in viewport width units, you multiply the viewport width (`vw`) by this ratio and add the height of the inner box (`6rem`)
I'm not sure if it's perfect, especially since we need to learn more about figuring out height ratios based on wireframes, but it was a lot better than what was happening before.